index.js

```diff
@@ -60,7 +60,8 @@ function handleArgs(command, args, options) {
 		options.cleanup = false;
 	}
 
-	if (process.platform === 'win32' && path.basename(command) === 'cmd.exe') {
+	const basename = path.basename(command);
```

Comment:
```suggestion
	const basename = path.basename(command, '.exe');
```

Then the next line does not need to be changed.